src/modules/0.0.23/aws_ecs_fargate/mappers/service.ts

```diff
@@ -33,8 +33,7 @@ export class ServiceMapper extends MapperBase<Service> {
     (a?.securityGroups?.every(asg => !!b?.securityGroups?.find(bsg => Object.is(asg.groupId, bsg.groupId))) ??
       false) &&
     Object.is(a?.subnets?.length, b?.subnets?.length) &&
-    (a?.subnets?.every(asn => !!b?.subnets?.find(bsn => Object.is(asn, bsn))) ?? false) &&
-    Object.is(a.forceNewDeployment, b.forceNewDeployment);
+    (a?.subnets?.every(asn => !!b?.subnets?.find(bsn => Object.is(asn, bsn))) ?? false);
```

Comment:
I think this is only checking if `a.subnets`⊂`b.subnets` which clearly does not yield equality. You might need to check if `b.subnets` also is a subset of `a.subnets` to make sure they're equal.

Reply:
i just copied the condition there... but we can do it as a followup, to isolate concepts of the commits